crates/ln-dlc-node/src/node/storage.rs

```diff
@@ -14,10 +14,9 @@ use lightning::chain::transaction::OutPoint;
 use lightning::ln::PaymentHash;
 use lightning::ln::PaymentPreimage;
 use lightning::ln::PaymentSecret;
+use parking_lot::Mutex;
```

Comment:
Can you explain why `parking_lot::Mutex` is better than `std::sync::Mutex`?

Reply:
A few reasons (arranged in order of relevance to our project):

- as some time ago we concluded that we don't care about the state of poisoning enough to warrant crashing the app, having a mutex that does not crash at such unlikely event is better for us
- it supports deadlock detection if we ever want to use it (I believe we turned it off recently, but we *can* turn it on)
- it's faster than the `std` counterpart